drydock/plugin.py

```diff
@@ -1,11 +1,52 @@
 from glob import glob
 import os
+import yaml
 import pkg_resources
 
+from tutor import env as tutor_env
 from tutor import hooks
 
 from .__about__ import __version__
 
+INSTALLATION_PATH = f'{os.getcwd()}/env'
+
+INIT_JOBS_SYNC_WAVE = -100
+
+
+@hooks.Actions.PLUGINS_LOADED.add()
+def get_init_tasks():
+    """Return the list of init tasks to run."""
+    init_tasks = list(hooks.Filters.CLI_DO_INIT_TASKS.iterate())
+
+    # Standarize deprecated COMMANDS_INIT Filter, to be removed in palm or later
+    standarized_commands = []
+    for service, init_path in list(hooks.Filters.COMMANDS_INIT.iterate()):
+        standarized_commands.append((service, tutor_env.read_template_file(*init_path)))
+
+    init_tasks.extend(standarized_commands)
+
+    templates_path = f'{INSTALLATION_PATH}/k8s/jobs.yml'
+    templates = list(yaml.load_all(open(templates_path, 'r', encoding='utf-8'), Loader=yaml.FullLoader))
+
+    response = []
+    for i, (service, definition) in enumerate(init_tasks):
+        for template in templates:
+            if template['metadata']['name'] == service + '-job':
+                data = dict()
```

Comment:
We can use that file to render the jobs and avoid [skip regular jobs](https://github.com/eduNEXT/drydock/blob/84ba87be34cf73e1b47712334874b4de45deb5e2/drydock/patches/kustomization#L12-L15), not seems like a patch to me because for example, the lms-job appears only one time in the jobs.yml, but we create some jobs with that template